flexmeasures/cli/data_delete.py

```diff
@@ -97,39 +97,36 @@ def delete_account(id: int, force: bool):
 @click.option(
     "--force/--no-force", default=False, help="Skip warning about consequences."
 )
-def delete_user_and_data(email: str, force: bool):
+def delete_a_user(email: str, force: bool):
     """
     Delete a user & also their assets and data.
     """
     if not force:
-        # TODO: later, when assets belong to accounts, remove this.
-        prompt = f"Delete user '{email}', including all their assets and data?"
+        prompt = f"Delete user '{email}'?"
```

Comment:
Line 75 could also use an update: `print(f"Deleting user {user} (and assets & data) ...")`